docs/sources/mimir/configure/configure-blocked-queries.md

```diff
@@ -30,6 +30,40 @@ To set up runtime overrides, refer to [runtime configuration]({{< relref "./abou
 The order of patterns is preserved, so the first matching pattern will be used.
 {{% /admonition %}}
 
+## Formatting queries to block
+
+Queries received by Mimir are parsed into PromQL expressions before blocking is applied.
+The `pattern` from the blocked queries is compared against Prometheus' string format representation of the parsed query,
+in order to allow consistent query blocking behavior regardless of formatting differences in the submitted queries.
+
+Among other transformations, the Prometheus formatter may reorder operators, remove empty selector braces,
+and eliminate newlines and extraneous whitespace.
+
+You can use Mimirtool's `promql format <query>` command to apply the Prometheus string formatting to a query
+to ensure your provided blocked query `pattern` is correct:
```

Comment:
I think this is the most important information for a reader. It should be the first part of the "Format queries" section so a reader can quickly understand if they care about the rest of the conceptual information and the task.